lib/src/js/analyze.js

```diff
@@ -43,7 +43,7 @@ function getBehaviors(results, filePath) {
     if (behavior.contentHref != filePath) continue;
 
     behaviors.push({
-      name: behavior.is,
+      name: behavior.is.replace('Polymer.', ''),
```

Comment:
Sometimes the behavior names start with `Polymer.`, this just strips that part out early. Previously it was being stripped out later on, and I forgot to also remove those.